### PR TITLE
https://issues.redhat.com/browse/ACM-7495 MSA GA 2.9

### PR DESCRIPTION
**Pull request by oafischer**

### CODE REVIEW

File: business_continuity/backup_restore/backup_msa.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@ When the activation data is restored on the new hub cluster, the restore control
 
 The automatic import feature using the Managed Service Account component is disabled by default. To enable the automatic import feature, complete the following steps:
 
-. Enable the Managed Service Account component by setting the `managedserviceaccount-preview` `enabled` parameter to `true` in the `MultiClusterEngine` resource. See the following example:
+. Enable the Managed Service Account component by setting the `managedserviceaccount` `enabled` parameter to `true` in the `MultiClusterEngine` resource. See the following example:
```
Comment (birsanv) [inline business_continuity/backup_restore/backup_msa.adoc:13]: If the MSA feature is enabled by default with MCE, you should remove this section (1) completely

**Review by birsanv — CHANGES_REQUESTED** [review]

File: business_continuity/backup_restore/backup_msa.adoc (hunk 1 above)
Comment (birsanv) [reply business_continuity/backup_restore/backup_msa.adoc:13]: ignore my comment; I misread one of the issues, MSA stays disabled by default

**Review by birsanv — APPROVED** [review]
lgtm

File: clusters/api/api_intro.adoc [hunk 2]
```diff
@@ -10,4 +10,4 @@ You can access the following APIs for cluster lifecycle management with the {mce
 * xref:../api/multicluster_engine.json.adoc#multiclusterengine-api[MultiClusterEngine API]
 * xref:../api/placement.json.adoc#placements-clusters-api[Placements API (v1beta1)]
 * xref:../api/placementdecision.json.adoc#placementdecisions-api[PlacementDecisions API (v1beta1)]
-* xref:../api/managed_serviceaccount.json.adoc#serviceaccount-api[Managed service account (Technology Preview)]
+* xref:../api/managed_serviceaccount.json.adoc#serviceaccount-api[Managed service account]
```
Comment (dockerymick) [inline clusters/api/api_intro.adoc]: I recommend that you update the title to read as camel-case to be consistent with the other API docs

File: clusters/api/managed_serviceaccount.json.adoc [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [#serviceaccount-api]
-= Managed service account (Technology Preview)
+= Managed service account
```
Comment (dockerymick) [inline clusters/api/managed_serviceaccount.json.adoc]:
```suggestion
= ManagedServiceAccount API
```

File: clusters/install_upgrade/adv_config_install.adoc [hunk 4]
```diff
@@ -139,14 +139,14 @@ metadata:
 spec:
   overrides:
     components:
-    - name: managedserviceaccount-preview
+    - name: managedserviceaccount
       enabled: true
 ----
 
-The `Managed-ServiceAccount` add-on can be enabled after creating `MultiClusterEngine` by editing the resource on the command line and setting the `managedserviceaccount-preview` component to `enabled: true`. Alternatively, you can run the following command and replace <multiclusterengine-name> with the name of your `MultiClusterEngine` resource.
+The `Managed-ServiceAccount` add-on can be enabled after creating `MultiClusterEngine` by editing the resource on the command line and setting the `managedserviceaccount` component to `enabled: true`. Alternatively, you can run the following command and replace <multiclusterengine-name> with the name of your `MultiClusterEngine` resource.
```
Comment (dockerymick) [inline clusters/install_upgrade/adv_config_install.adoc]:
```suggestion
The `ManagedServiceAccount` add-on can be enabled after creating `MultiClusterEngine` by editing the resource on the command line and setting the `managedserviceaccount` component to `enabled: true`. Alternatively, you can run the following command and replace <multiclusterengine-name> with the name of your `MultiClusterEngine` resource.
```

**Review by dockerymick — APPROVED** [review]
Made some suggestions, thanks Oliver!

**Pull request merged**